src/views/Settings/index.vue

```diff
@@ -68,12 +69,45 @@ export default {
       try {
         this.isLoading = true;
         const sectors = await Sector.list();
+        this.nextPage = sectors.next;
         this.sectors = sectors.results;
+        this.next = sectors.next;
         this.isLoading = false;
       } catch (error) {
         this.isLoading = false;
       }
     },
+
+    async listMoreSectors() {
+      if (this.isLoading || !this.nextPage) {
+        return;
+      }
+
+      try {
+        this.isLoading = true;
+        const newSectors = await Sector.list(this.nextPage);
+        this.nextPage = newSectors.next;
+        this.sectors = [...this.sectors, ...newSectors.results];
+        if (newSectors) {
+          this.isLoading = false;
+        }
+      } catch (error) {
+        this.isLoading = false;
+      }
+    },
+
+    onScroll() {
+      if (
+        this.$refs.sectorsSection.scrollTop + this.$refs.sectorsSection.clientHeight >=
+        this.$refs.sectorsSection.scrollHeight
```

Comment:
To make the code more readable, you could put these values (`this.$refs.sectorsSection...`) in variables.
At the same time, assign the calc to a variable to name what you expect to achieve with it, such as "isScrollInBottom"